fastlane/lib/fastlane/actions/oclint.rb

```diff
@@ -12,8 +12,18 @@ def self.run(params)
           UI.user_error!("You have to install oclint or provide path to oclint binary. Fore more details: ") + "http://docs.oclint.org/en/stable/intro/installation.html".yellow
         end
 
+        UI.user_error!("Could not find json compilation database at path '#{params[:compile_commands]}'") unless File.exist?(params[:compile_commands])
         compile_commands = params[:compile_commands]
```

Comment:
a little nitpicky - but can this line go above and use just `compile_commands` in the string interpolation and `File.exists?` check? just to not add the `params[]` into the `user_error` line?